koku/api/report/test/util/model_bakery_loader.py

```diff
@@ -396,9 +397,17 @@ def load_openshift_data(self, cluster_id, on_cloud=False):
         with OCPReportDBAccessor(self.schema) as accessor:
             accessor.populate_pod_label_summary_table(report_period_ids, self.first_start_date, self.last_end_date)
             accessor.populate_volume_label_summary_table(report_period_ids, self.first_start_date, self.last_end_date)
-            accessor.update_line_item_daily_summary_with_enabled_tags(
-                self.first_start_date, self.last_end_date, report_period_ids
-            )
+            # remove disabled tag keys
+            sql_params = {
+                "start_date": self.first_start_date,
+                "end_date": self.last_end_date,
+                "report_period_ids": report_period_ids,
+                "schema": self.schema,
+            }
+            sql = pkgutil.get_data("api.report.test.util.mock_trino_sql", "ocp/remove_disabled_tags.sql")
+            sql = sql.decode("utf-8")
+            table_name = accessor._table_map["line_item_daily_summary"]
+            accessor._prepare_and_execute_raw_sql_query(table_name, sql, sql_params)
```

Comment:
I don't understand the purpose of this change. Why not just leave this functionality where it is and just update the names and the docstring to indicate that it is used for unit testing? The purpose of the accessor is to run sql against the db, so we should leave that functionality there.

Reply:
I just sent a message to the team chat about this 😂 The reason for the change is that I would like to disconnect these files from our data processing flow. I want to make clear that the only purpose this file serves now is for unit testing.

Reply:
I am open to suggestions though, if we want to keep it in the accessor that is fine. I just want a cleaner way of understanding which sql files are being used for unit-testing purposes only. But that could just mean moving them to a separate directory.